File: frontend/src/types/community-answers-report/select-community-answers-report-dto.ts
import { BaseSelectCommunityReportDTO } from "../community-content-report/base-select-community-report-dto";
import { SelectReportedAnswerDTO } from "../community-content-report/select-reported-content-dto";

export interface SelectCommunityAnswersReportDTO extends BaseSelectCommunityReportDTO {
    answerId: string;
    answer?: SelectReportedAnswerDTO | null;
}

File: platform-core-service/platform-core-service.Common/Models/DTOs/EntityDTO/CommunityAnswersReport/SelectCommunityAnswersReportDTO.cs
using platform_core_service.Common.Models.DTOs.EntityDTO.CommunityContentReport;

namespace platform_core_service.Common.Models.DTOs.EntityDTO.CommunityAnswersReport
{
    public class SelectCommunityAnswersReportDTO : BaseSelectCommunityReportDTO
    {
        public string AnswerId { get; set; } = null!;
        public SelectReportedAnswerDTO? Answer { get; set; }
    }
}
